packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/edit/vsphere-user-edit.controller.js

```diff
@@ -54,4 +73,8 @@ export default class {
         );
       });
   }
+
+  isEdited() {
+    return JSON.stringify(this.pickUserInformationToSend()) !== this.originUser;
```

Comment:
you could also use `angular.equals` to avoid doing `JSON.stringify`. To check changes, we generally use that pattern

```suggestion
    return !angular.equals(this.pickUserInformationToSend(), this.originUser);
```